src/common/components/cards/card-kebab-menu-button.tsx

```diff
@@ -39,12 +37,12 @@ export interface CardKebabMenuButtonProps {
 }
 
 export class CardKebabMenuButton extends React.Component<CardKebabMenuButtonProps, CardKebabMenuButtonState> {
+    private toastRef: React.RefObject<Toast>;
     constructor(props: CardKebabMenuButtonProps) {
         super(props);
-
+        this.toastRef = React.createRef();
         this.state = {
             showNeedsSettingsContent: false,
-            showingCopyToast: false,
             toastText: '',
```

Comment:
`toastText` here is obsolete now, right?

Reply:
yes, forgot to remove